packages/slate-react/src/components/editor.js

```diff
@@ -44,6 +44,13 @@ class Editor extends React.Component {
     placeholder: Types.any,
     plugins: Types.array,
     readOnly: Types.bool,
+    renderAnnotation: Types.func,
+    renderBlock: Types.func,
+    renderDecoration: Types.func,
+    renderDocument: Types.func,
+    renderEditor: Types.func,
+    renderInline: Types.func,
+    renderMark: Types.func,
```

Comment:
Need to add another property `decorateNode: Types.func,`.
And I would advise moving the list to constants `/packages/slate-react/src/constants/...`

Reply:
I put the fix as part of constants/event-handlers.js since they all share "Types.func". Or we create a different file for it?